src/bindings/c/src/CMakeLists.txt

```diff
@@ -14,7 +14,8 @@ add_library(openvino::runtime::c ALIAS ${TARGET_NAME})
 target_link_libraries(${TARGET_NAME} PRIVATE openvino)
 
 target_include_directories(${TARGET_NAME} PUBLIC
-    $<BUILD_INTERFACE:${OpenVINO_C_API_SOURCE_DIR}/include>)
+    $<BUILD_INTERFACE:${OpenVINO_C_API_SOURCE_DIR}/include/>
+    $<BUILD_INTERFACE:${OpenVINO_C_API_SOURCE_DIR}/include/openvino>)
```

Comment:
```suggestion
    $<BUILD_INTERFACE:${OpenVINO_C_API_SOURCE_DIR}/include/>)
```

Reply:
I just wan to set separated legacy api header and 2.0 C api header, ok let's merge them together.

Reply:
done